### PR TITLE
59 design UI for browsing and purchasing in marketplace

### PR DESCRIPTION
**Pull request by kivine99**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -83,9 +83,11 @@ function App() {
...
 function AppWrapper() {
   return (
-    <Router>
-      <App />
-    </Router>
+    <CurrencyProvider>
+      <Router>
+        <App />
+      </Router>
+    </CurrencyProvider>
```
Comment (VanJackal) [inline src/App.tsx:90]: this feels a bit odd to me, if at some point we wanted another global provider/context how hard is it for us to add it

File: src/pages/Vendors/components/RequiredOfferItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        ) : (
+          <CurrencyComp
+            amount={requiredCurrency.coins}
+            scale={CURRENCY_SCALE}
+            style={{
+              color: AMOUNT_NOT_MET_COLOUR,
+            }}
+          />
+        )
+      );
```
Comment (VanJackal) [inline src/pages/Vendors/components/RequiredOfferItem/index.tsx:92]: maybe move the ternary into the color field (everything looks like it stays the same)

File: src/App.tsx (hunk 1 above)
Comment (kivine99) [reply src/App.tsx:90]: You can just keep wrapping it in contexts

Comment (VanJackal) [reply src/App.tsx:90]: I feel like that could get ugly later but we should deal with that if it becomes an issue

**Pull request merged**